Content.Shared/Forensics/SharedForensicScannerComponent.cs

```diff
@@ -0,0 +1,6 @@
+namespace Content.Shared.Forensics
+{
+    public abstract class SharedForensicScannerComponent : Component
+    {
+    }
+}
```

Comment:
do you need a shared component? doesn't look like you do anything with it clientside